build.sh

```diff
@@ -47,8 +50,10 @@ install_rpms() {
     # development version.
     builddeps=$(grep -v '^#' "${srcdir}"/src/build-deps.txt)
 
+    # Freeze kernel due to https://github.com/coreos/coreos-assembler/issues/2003
+    frozendeps=$(echo https://kojipkgs.fedoraproject.org//packages/kernel/5.9.16/200.fc33/"${arch}"/kernel-{core,modules}-5.9.16-200.fc33."${arch}".rpm)
```

Comment:
Future cleanup: Use `koji download-build` and also split this to a separate `src/pinned-deps.txt` or so.

Reply:
(It's basically a lockfile just like we're using for rpm-ostree and maybe at some point it will make sense to push that into libdnf and have it usable for `/usr/bin/dnf` in containers too)

Reply:
Hmm, actually now with the archive repo Dusty set up, it would've been enough to add that and use the NEVR directly.